modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/portlet/action/CopyFileEntryMVCRenderCommand.java

```diff
@@ -45,10 +112,51 @@ protected String getPath() {
 		return "/document_library/copy_file_entry.jsp";
 	}
 
+	private void _sendRedirect(
+			RenderRequest renderRequest, RenderResponse renderResponse)
+		throws PortletException {
+
+		try {
+			HttpServletResponse httpServletResponse =
+				_portal.getHttpServletResponse(renderResponse);
+
+			httpServletResponse.sendRedirect(
+				ParamUtil.getString(renderRequest, "redirect"));
+		}
+		catch (IOException ioException) {
+			throw new PortletException(ioException);
+		}
+	}
+
+	private void _validateCopyToSize(
+			long companyMaxSizeToCopy, long groupMaxSizeToCopy,
+			long systemMaxSizeToCopy, long fileEntrySize)
+		throws FileEntrySizeLimitExceededException {
+
+		if (!DLCopyValidationUtil.isCopyToAllowed(
+				companyMaxSizeToCopy, groupMaxSizeToCopy, systemMaxSizeToCopy,
+				fileEntrySize)) {
+
+			throw new FileEntrySizeLimitExceededException();
+		}
+	}
```

Comment:
Instead of passing the three max sizes, why not use the configuration values directly? It is a bit pointless to pass as parameters things that are already in scope.